main.py

```diff
@@ -1145,3 +1164,59 @@ def handle_link_maintenance_end(self, event):
             link.endpoint_a.enable()
             link.endpoint_b.enable()
             self.notify_link_status_change(link, reason='maintenance')
+
+    @listen_to('topology.interruption.start')
+    def on_interruption_start(self, event: KytosEvent):
+        """Deals with the start of service interruption."""
+        with self._links_lock:
+            self.handle_interruption_start(event)
+
+    def handle_interruption_start(self, event: KytosEvent):
+        """Deals with the start of service interruption."""
```

Comment:
@Ktmi considering the point that @italovalcy has also brought to the attention on yesterday's meeting, maybe on both `handle_interruption_start` and `handle_interruption_end` it would good points to centralize `log.info` entries, after all, we're also using `log.info` for port status changes, so here it'd reasonable too. Another benefit of centralizing here on `topology` is that it could simplify certain logs for interruption sources. Wdyt?

Reply:
@Ktmi let me know about this one here, if you agree and also if it'll be part of this PR or a subsequent one.